### PR TITLE
Fix sentry usage & report grcov errors.

### PR DESCRIPTION
**Pull request by La0**
Needed for https://github.com/mozilla/grcov/pull/304

### CODE REVIEW

**Comment by La0** [comment]
Build is failing due to missing grcov 0.5.2 (that's normal).

@calixteman is there a simple way to do a grcov RC ?

**Comment by La0** [comment]
We now have a [dedicated project](https://sentry.prod.mozaws.net/operations/code-coverage/) on Mozilla sentry. 
I updated the runtime secrets to use that project instead of release-services

File: bot/code_coverage_bot/utils.py [hunk 1]
```diff
@@ -69,12 +69,20 @@ def run_check(command, **kwargs):
         output, error = proc.communicate()
 
     if proc.returncode != 0:
-        log.info(
+        if isinstance(output, bytes):
```
Comment (marco-c) [inline bot/code_coverage_bot/utils.py]: Nit: these should always be bytes

File: tools/code_coverage_tools/log.py [hunk 2]
```diff
@@ -42,24 +44,20 @@ def setup_papertrail(project_name, channel, PAPERTRAIL_HOST, PAPERTRAIL_PORT):
...
-        # tags=...
-        # repos=...
-    )
+    # Detect environment
+    if "TASK_ID" in os.environ:
+        site = "taskcluster"
+    elif "DYNO" in os.environ:
+        site = "heroku"
+    else:
+        site = "unknown"
```
Comment (marco-c) [inline tools/code_coverage_tools/log.py]: This would be local, right?

Comment (La0) [reply tools/code_coverage_tools/log.py]: Yeap

**Pull request merged**